src/controllers/AlunoController.js

```diff
@@ -0,0 +1,129 @@
+import Aluno from '../models/Aluno';
+import Photo from '../models/photo';
+
+class AlunoController {
+  // Index
+  async index(req, res) {
+    const alunos = await Aluno.findAll({
+      attributes: ['id', 'nome', 'sobrenome', 'email', 'idade', 'peso', 'altura'],
+      order: [['id', 'DESC'], [Photo, 'id', 'DESC']],
+      include: {
+        model: Photo,
+        attributes: ['filename', 'originalname', 'url_photo'],
+      },
+    });
+    res.json(alunos);
+  }
+
+  // Store
+  async store(req, res) {
```

Comment:
em CRUDS o padrão usado para salvar um recurso geralmente é `create`. Quando eu vi essa função eu primeiramente pensei que era algo relacionado ao `store` do redux, mas depois de ler eu vi que é a ação de persistir.
```suggestion
  async create(req, res) {
```